mlrun/datastore/base.py

```diff
@@ -217,6 +232,7 @@ def reader(*args, **kwargs):
                     ):
                         raise ex
 
+                    #  TODO ML-6308 fix.
```

Comment:
```suggestion
                    # TODO: fix ... (ML-6308)
```
and replace ... with an explanation.

I'm not sure I understand what ML-6308 says though. Is it even a proven bug?